src/bot.py

```diff
@@ -35,8 +35,11 @@
     ]
 )
 
+mininterval = config['mininterval']
+maxinterval = config['maxinterval']

```

Comment:
The appropriate changes that need to be made in the [config.py](src/config.py) don't seem to be there.

Reply:
See #5.